### PR TITLE
fix: apply `recurrenceRule` changes (fix #936)

### LINKED ISSUE #936: updateSchedule method does not refresh recurrenceRule and attendees fields
## Describe the bug
I am trying to use updateSchedule method to refresh recurrenceRule and attendees fields. But it doesn't work.

calendar.updateSchedule(schedule.id, schedule.calendarId, {
    recurrenceRule: 'repeated events',
    attendees: ['David Perez', 'Nicole Oliver']
});

After that I use the method getSchedule: calendar.getSchedule(schedule.id, schedule.calendarId); and the recurrenceRule and attendees fields are still empty.
{
   ...
   recurrenceRule: '',
   attendees: [],
   ....
}

And the display of the icon for the event on the calendar also does not change.

Also, for example, if I create a schedule, I set a field (recurrenceRule: 'repeated events' /  attendees: ['David Perez', 'Nicole Oliver']) when it is updated to an empty string / array using the updateSchedule method, nothing happens.

If I try to save information in this fields: title, body, start, end,  isAllday, category, location, state, isPrivate, isPending, isFocused, isReadOnly, raw, color, bgColor, borderColor; It works. But if I try to save empty string, for example, in location, title ... fields it doesn't work.

I checked and no changes are saved for: goingDuration, comingDuration, dueDateClass, isVisible, dragBgColor, customStyle.

## To Reproduce
1. Update schedule fields recurrenceRule and attendees by using updateSchedule method.
2. Check if the field (recurrenceRule, attendees) values ​​have changed when using the method getSchedule.
3. Check if the icons for the displayed event on the calendar have changed.

## Expected behavior
It should be possible to change any property of a schedule with the method updateSchedule.

## Desktop (please complete the following information):
 - OS: Mac OS
 - Browser: FireFox, Chrome
 - tui.calendar [1.15.1]

### PR DESCRIPTION
**Pull request by adhrinae**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

- This PR applies the `recurrenceRule` changes in the `updateSchedule` method.

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/js/controller/base.js [hunk 1]
```diff
@@ -309,6 +309,10 @@ Base.prototype.updateSchedule = function(schedule, options) {
         schedule.set('attendees', options.attendees);
     }
 
+    if (options.recurrenceRule) {
```
Comment (lja1018) [inline src/js/controller/base.js:312]: 따로 문자열인지 체크는 안해도 되는건가요?

**Review by lja1018 — COMMENTED** [review]
리뷰 완료합니다.

File: src/js/controller/base.js (hunk 1 above)
Comment (adhrinae) [reply src/js/controller/base.js:312]: 사실 그런걸 고려하기 시작하면 이 메서드는 싹 다 뜯어고쳐야해서 😢 
그냥 기존에 다른 속성에 하던 방식과 동일한 방식으로 했습니다.

**Review by jwlee1108 — APPROVED** [review]
리뷰 완료합니다.

**Pull request merged**